examples/with-firebase-authentication/.env.local.example

```diff
@@ -0,0 +1,13 @@
+# Update these with your Firebase app's values.
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=my-example-app-id
+NEXT_PUBLIC_FIREBASE_CLIENT_EMAIL=my-example-app-email@example.com
+NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY=MyExampleAppAPIKey123
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=my-example-app.firebaseapp.com
+NEXT_PUBLIC_FIREBASE_DATABASE_URL=https://my-example-app.firebaseio.com
+
+# Your Firebase private key.
+NEXT_PUBLIC_FIREBASE_PRIVATE_KEY=some-key-here
```

Comment:
This key is probably private too, right? let's remove the `NEXT_PUBLIC_` prefix from here too, and same for any other Firebase key that's only used in the server.